lib/aadutils.js

```diff
@@ -265,3 +265,62 @@ exports.concatUrl = (url, rest) => {
   return rest ? url.concat(hasParam ? '&' : '?').concat(rest.join('&')) : url;
 };
 
+
+// This is a list maintained by the AAD server team, will need to keep an eye on this
+// as things change.  Not ideal, but it is what it is
+//n general, a change like this (adding a new cookie attribute)
+// should be backward compatible as RFC 6265 specifies that browsers should
+// ignore unknown cookie attributes. However, for the specific case of the
+// SameSite attribute, some browsers incorrectly implemented the attribute or
+//  implemented an earlier draft which had contradictory behavior.
+// For these browsers which attempted to support SameSite,
+// but have bugs in their support, we want to omit the SameSite=None attribute.
```

Comment:
Add a link here to the Chromium team guidance so that people can get check it against the official guidance? https://www.chromium.org/updates/same-site/incompatible-clients

Reply:
will add

Reply:
updated